tests/config.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import annotations
+
+checkout_blocks_and_plots = True
```

Comment:
This file is only applicable if using the matrix generation script in blockchain.  This file should be removed.